editions/tw5.com/tiddlers/filters/allafter.tid

```diff
@@ -0,0 +1,17 @@
+caption: allafter
+created: 20151017145021839
+creator: matabele
```

Comment:
OK -- seen. I am also inquiring with respect the general case. To modify code, I now submit a new pull request from a new branch?